lochness/models/data_sinks.py

```diff
@@ -113,10 +113,10 @@ def get_all_data_sinks(config_file: Path, active_only: bool = False) -> List["Da
 
     @staticmethod
     def get_matching_data_sink(config_file: Path,
-                               data_sink_name: str,
                                site_id: str,
                                project_id: str,
                                active_only: bool = False,
+                               data_sink_name: str = None,
```

Comment:
```suggestion
                               data_sink_name: Optional[str] = None,
```

Prefer using Optional typing for types that can be None